### ISSUE TITLE
[Question]: 蒸馏后的模型使用问题

### ISSUE THREAD

**Issue by Drizzlenum** [PaddlePaddle/PaddleNLP#3835]
### 请提出你的问题

正常生成的模型，经过TaskFlow实例化后可以进行多次预测，按照模型蒸馏过程，蒸馏后的模型实例化后只有第一次预测样例正确，第二次及以后的预测会出现不断按字数补全样本的效果，结果有误。故在模型使用过程中每次都重新实例化，这样推测时间就加长了，不知道具体是什么原因。麻烦各位解答。

**Comment by linjieccc** [comment]
Hi，

数据蒸馏部分的代码有做改动吗，另外方便提供下训练数据或者可复现问题的最小数据集合么

**Comment by Drizzlenum** [comment]
> 
抱歉数据不方便提供，以下是我的蒸馏过程使用的命令
python doccano.py --doccano_file ./data/08161.json --task_type ext --save_dir ./data1 --splits 0.8 0.1 0.1

python finetune.py --device gpu  --logging_steps 10 --save_steps 100 --eval_steps 100  --seed 42 --model_name_or_path uie-base --output_dir $finetuned_model --train_path data/train.txt --dev_path data/dev.txt   --max_seq_length 512   --per_device_eval_batch_size 4 --per_device_train_batch_size  4 --num_train_epochs 100 --learning_rate 1e-5 --label_names 'start_positions' 'end_positions' --do_train --do_eval --do_export --export_model_dir $finetuned_model --overwrite_output_dir --disable_tqdm True --metric_for_best_model eval_f1 --load_best_model_at_end  True --save_total_limit 1 

python data_distill.py --data_path ../data --save_dir student_data --task_type entity_extraction --synthetic_ratio 10 --model_path ../checkpoint/model_best

python evaluate_teacher.py --task_type entity_extraction --test_path ./student_data/dev_data.json --label_maps_path ./student_data/label_maps.json --model_path ../checkpoint/model_best/

python train.py --task_type entity_extraction --train_path student_data/train_data.json --dev_path student_data/dev_data.json --label_maps_path student_data/label_maps.json --num_epochs 2000 --encoder ernie-3.0-mini-zh --batch_size 4 --learning_rate 1e-5 --seed 42

**Comment by linjieccc** [comment]
@Drizzlenum 你的paddle版本是多少？

**Comment by Drizzlenum** [comment]
@linjieccc paddlepaddle-gpu=2.4.0，paddlenlp是2.4.2

**Comment by Drizzlenum** [comment]
@linjieccc 已解决，是和版本有关，paddlenlp需要是2.4.3
